src/storage/txn/commands/cleanup.rs

```diff
@@ -0,0 +1,33 @@
+use crate::storage::metrics::{self, KV_COMMAND_COUNTER_VEC_STATIC};
+use crate::storage::txn::commands::{Command, CommandExt, TypedCommand};
+use crate::storage::txn::latch::{self, Latches};
+use crate::storage::{Context, TimeStamp};
+use crate::{command, ctx, gen_lock, tag, ts, write_bytes};
+use std::fmt::{self, Debug, Display, Formatter};
+use std::iter;
+use txn_types::Key;
+
+command! {
+    /// Rollback mutations on a single key.
```

Comment:
Can you move these comments to inner module comments for each command please?

Reply:
If we move these comments to inner module, these comments won't showup on the document web page built by `cargo doc`.
Does that matter?

Reply:
They should show up as docs for the module. Do they not?

Reply:
> They should show up as docs for the module. Do they not?

No because thouse modules are not `pub`.

Reply:
Ah, ok, just leave them as is then.